src/standalone/env_test.go

```diff
@@ -31,16 +31,84 @@ func TestNewEnv(t *testing.T) {
 		assert.NotEmpty(t, env.K8PodUID)
 		assert.NotEmpty(t, env.K8BasePodName)
 		assert.NotEmpty(t, env.K8Namespace)
+		assert.NotEmpty(t, env.K8ClusterID)
 
 		assert.NotEmpty(t, env.WorkloadKind)
 		assert.NotEmpty(t, env.WorkloadName)
 
 		assert.True(t, env.OneAgentInjected)
 		assert.True(t, env.DataIngestInjected)
 	})
+	t.Run(`create new env for only data-ingest injection`, func(t *testing.T) {
```

Comment:
Should we have also a test with no injection at all?

Reply:
not really, as this should only be run if at least 1 mutation was done.
Without injection there shouldn't be any init-container added so the standalone init shouldn't run